fili-core/src/main/java/com/yahoo/bard/webservice/web/FiliResponseWriter.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+
+/**
+ * Default writer for response. Will choose to use the writer type from format type.
+ */
+public class FiliResponseWriter implements ResponseWriter {
+
+    private final FiliResponseWriterSelector responseWriterSelector = new FiliResponseWriterSelector();
+
+    /**
+     * Selects a writer from mapping and writes response.
+     *
+     * @param request  ApiRequest object with all the associated info in it
+     * @param responseData  Data object containing all the result information
+     * @param os  OutputStream
+     *
+     * @throws IOException if a problem is encountered writing to the OutputStreamC
+     */
+    public void write(
+            ApiRequest request,
+            ResponseData responseData,
+            OutputStream os
+    ) throws IOException {
+        ResponseWriter writer = responseWriterSelector.select(request);
+        writer.write(request, responseData, os);
```

Comment:
We should do a null check on `writer` here and throw a useful error message if the response type isn't in the map (i.e. isn't recognized). Otherwise, if at some point down the line, someone adds a custom response format, but forgets to register the writer, then they'll just get a vague `NullPointerException`.